### PR TITLE
Add front-office Group link and Survey link to admin

### PR DESCRIPTION
**Pull request by cveneziani**
- Also add logo image to Group
- Also don't load rails rumble context on admin pages

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -6,10 +6,14 @@ class ApplicationController < ActionController::Base
   rescue_from Errors::UnauthorizedError,  with: :render_not_found
   rescue_from ::Groups::NotFoundError,    with: :render_not_found
 
-  before_action :rails_rumble_context
+  before_action :rails_rumble_context, unless: :admin_controller?
```
Comment (kdisneur) [inline app/controllers/application_controller.rb]: Do we still need the rails rumble context or could we totally get rid of it?

Comment (cveneziani) [inline app/controllers/application_controller.rb]: We can get rid of it but I think it should be done in a different PR.
So for now, I'm just disabling it for admin :grimacing:

**Comment by kdisneur** [comment]
:+1:

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (kdisneur) [inline app/controllers/application_controller.rb]: ok - make sense :)

**Comment by alaibe** [comment]
:+1:
